common/components/common/projects/EditProjectForm.jsx

```diff
@@ -324,11 +325,13 @@ class EditProjectForm extends React.PureComponent<Props,State> {
             }
           </label>
           <div className="character-count">
```

Comment:
Let's put the character counts below the fields and to the right per the design.